components/BackdropMap.tsx

```diff
@@ -12,25 +12,19 @@ export default function BackdropMap() {
     const [zoom, setZoom] = useState(9)
 
   useEffect(() => {
```

Comment:
A cleanup function here could look something like:

```ts
return () => map.current!.remove()
```

The idea being that the side effect captured by this useEffect hook is the creation of this Mapbox `Map` object. Thankfully, `Map` comes with a `.remove()` method which tears down the map and cleans it all up, so our cleanup function can just leverage that.